### PR TITLE
chore: minor CONTRIBUTING update

### PR DESCRIPTION
**Pull request by ddysher**
**What this PR does / why we need it**:

R.T

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @kdada 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
 
-First, you can fork this repository via the `Fork` button on the top of this page (If you like Nirvana, please star it :).  
-After that, you can immediately clone `https://github.com/<your-account-name>/nirvana` to your local path `$GOPATH/src/github.com/caicloud/nirvana`.  
-There are some environment requirements:  
+First, you can fork this repository via the `Fork` button on the top of this page (If you like Nirvana, please star it :)).
+Then you can clone Nirvana to your local path:
+
+```
+$ mkdir -p $GOPATH/src/github.com/nirvana
+$ go get github.com/caicloud/nirvana
```
Comment (kdada) [inline CONTRIBUTING.md]: Why do we need to `mkdir` first?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
+
+# Avoid push to caicloud/nirvana
+$ git config push.default nothing
+
+$ git remove add $USER git@github.com:$USER/nirvana.git
```
Comment (kdada) [inline CONTRIBUTING.md]: `$USER` is the os user which may be not equal to the github account.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ddysher) [reply CONTRIBUTING.md]: typo

File: CONTRIBUTING.md (hunk 2 above)
Comment (ddysher) [reply CONTRIBUTING.md]: this is just copy/paste from kubernetes & cyclone ..

**Comment by ddysher** [comment]
@kdada updated

**Comment by kdada** [comment]
/lgtm

But the bot is on holiday.

**Comment by ddysher** [comment]
he's back

**Pull request merged**